internal/npm_install/generate_build_file.ts

```diff
@@ -60,6 +60,11 @@ const BAZEL_VERSION = args[6];
 const PUBLIC_VISIBILITY = '//visibility:public';
 const LIMITED_VISIBILITY = `@${WORKSPACE}//:__subpackages__`;
 
+const isModuleRegExp = /^export|^import(?!declare globa)/m;
```

Comment:
intentional misspelling `globa` instead of `global`?

Reply:
Unintentional